concepts/nil/about.md

```diff
@@ -1,65 +1,12 @@
 # About
 
-## Key Points:
-
-- When a class' `.new` method is called to create an object instance, the `.initialize` method is passed all arguments to initialize the instance's state.
-- instance variable names are prefixed with `@`.
-- instance variables default to `nil` until they are explicitly set.
-- instance variables are private by default, and they should be manipulated with getters and setters
+[Nil][nil-dictionary] is an English word meaning "nothing" or "zero". In Ruby, `nil` is used to express the _absence_ of an object. In other programming languages, `null` or `none` values may play a similar role.
 
 ```ruby
-class Backpack
-  initialize(owner)
-    @owner = owner
-  end
-
-  def owner
-    @owner
-  end
-
-  def owner=(new_owner)
-    @owner = new_owner
-  end
-end
+# I do not have a favorite color
+favorite_color = nil
```

Comment:
This could be better represented as "no  favorite color collected" while if they really have no favorite color and we have collected that information, "none" might be a better representation, rather than `nil`.

This is already merged, though, so just a note.